packages/trip-form/src/TripOptions/FeaturedOptionOverlay.tsx

```diff
@@ -0,0 +1,71 @@
+import { ExternalLinkAlt } from "@styled-icons/fa-solid/ExternalLinkAlt";
+import React, { FunctionComponent } from "react";
+
+import * as S from "./styled";
+import { Company, Modes } from "./types";
+import { getCategoryModes, getCategoryPrimaryMode } from "./util";
+
+const FeaturedOptionOverlay = ({
+  featuredOption,
+  setFeaturedOption,
+  supportedModes,
+  CompanyIcon,
+  DetailedModeIcon
```

Comment:
No. I put these components at the bottom because they relate to each other more than the other props. I could sort them if you like, but in my view that makes them more difficult to find.